protocol/lib/ante/app_injected_msg_ante_wrapper_test.go

```diff
@@ -86,9 +86,6 @@ func TestValidateMsgType_AppInjectedMsg(t *testing.T) {
 		}
 	}
 
-	// (2 * num of "addAppInjectedMsg=true" * num of app-injected msgs) + num of "addAppInjectedMsg=false")
-	// (2 * 1 * 5) + 3 = 11
-	require.Len(t, allTestCases, 13)
```

Comment:
Removing all asserts like this because they don't provide much value and aren't maintainable. (Everyone that breaks this will just increment or decrement to make it pass becuase they can't verify the full list. So it doesn't do anything but slow people down.)